lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js

```diff
@@ -120,7 +120,17 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
         
         return courseId;
     },
-            
+
+    _replaceKafUriWithToken: function(sourceUrl, kafUri, kalturaToken) {
+        sourceUrl = this._removeProtocolFromUrl(sourceUrl);
+        kafUri = this._removeProtocolFromUrl(kafUri);
+        return sourceUrl.replace(kafUri, kalturaToken);
```

Comment:
why not a single PR that includes the DB-insert as well?

as for the removal of the http and it being added later - OK.